Dockerfile

```diff
@@ -1,28 +1,32 @@
-FROM ruby:2.7.2
+FROM ruby:2.7.6
 
-# throw errors if Gemfile has been modified since Gemfile.lock
+# Throw errors if Gemfile has been modified since Gemfile.lock
 RUN bundle config --global frozen 1
 
-RUN mkdir -p /usr/src/app
+# Set up the app directory
 WORKDIR /usr/src/app
 
-COPY Gemfile /usr/src/app/
-COPY Gemfile.lock /usr/src/app/
-RUN bundle install
+# Configure bundler
+ENV LANG=C.UTF-8 \
+    BUNDLE_JOBS=4 \
+    BUNDLE_RETRY=3
 
-COPY . /usr/src/app
+# Store Bundler settings in the project's root
```

Comment:
This is a really small distinction, but from what I can tell: 

The next line doesn't so much store the bundler settings as _tell bundler_ where it's settings are stored. We're only setting a runtime env variable here.

I am also somewhat curious if this line is needed. Based on the [`bundler` documentation](https://bundler.io/man/bundle-config.1.html) it looks like `bundler` will automatically look for a config file at:
```
Local config (<project_root>/.bundle/config or $BUNDLE_APP_CONFIG/config)
```

Not sure what counts as the `project_root` in this case. Would that be the root directory of the docker image/where we run `bundle exec` from? 

At any rate, nothing to change here, just learning more about the set up. 👍